package.json

```diff
@@ -24,6 +24,7 @@
     "babel-preset-react": "^6.23.0",
     "babel-register": "^6.7.2",
     "body-parser": "^1.17.2",
+    "bookshelf": "^0.12.1",
     "dotenv": "^4.0.0",
```

Comment:
I don't see bookshelf being used anywhere. If it wasn't used, please uninstall.

Reply:
done